src/login/components/GroupManagement/InviteRoleCheckboxes.js

```diff
@@ -7,13 +7,25 @@ const checkboxNamesLabels = [
   { owner: "User can edit who belongs to the group as an owner" },
 ];
 
-export const RenderCheckboxInput = ({ meta, input, id, label, type }) => {
+export const RenderCheckboxInput = ({
+  meta,
+  input,
+  id,
+  label,
+  type,
+  disabled,
+}) => {
   const { error } = meta;
   const { onChange, value } = input;
+  const checkedStyling = value ? "checked" : null;
+  const disabledStyling = disabled ? "disabled" : null;
   return (
     <Fragment>
       <div className="checkbox-label">
-        <label className={value ? "checked" : null} htmlFor={id}></label>
+        <label
+          className={`${checkedStyling} ${disabledStyling}`}
```

Comment:
I think that using classname "null" might be misleading,
This is due to the classname changing to "null" not an empty classname

My suggestion is:
For the "checkedStyling" could we change from "null" to "unchecked" 
For "disabledStyling" could we change from "null" to "abled"